modules/dag-l1/src/main/scala/org/tessellation/dag/l1/StateChannel.scala

```diff
@@ -121,7 +121,7 @@ class StateChannel[
   private val runConsensus: Pipe[F, BlockConsensusInput, FinalBlock] =
     _.evalTap(input => logger.debug(s"Received block consensus input to process: ${input.show}"))
       .evalMap(
-        new BlockConsensusCell[F](_, blockConsensusContext)
+        new BlockConsensusCell[F](_, blockConsensusContext, storages.lastSnapshot.getOrdinal)
```

Comment:
if you don't know the last snapshot, the consensus is forbidden and should never happen (to make a consensus, you need to know tips, and tips are taken from the latest snapshot). correct me if I'm wrong @TheMMaciek